godtools/App/Share/Data/MobileContentAuthTokenRepository/Cache/MobileContentAuthTokenCache.swift

```diff
@@ -29,8 +29,25 @@ class MobileContentAuthTokenCache {
         }
     }
     
-    func getAuthToken() -> String? {
+    func getAuthToken(for userId: String) -> String? {
         
-        return keychainAccessor.getMobileContentAuthToken()
+        return keychainAccessor.getMobileContentAuthToken(userId: userId)
+    }
+    
+    func storeUserId(_ userId: String) {
```

Comment:
I would suggest forcing both the userId and authToken to be stored together since those are paired together rather than having separate methods to store each.  Having separate methods to store each could cause inconsistencies if one value were stored and another wasn't.